src/material-experimental/mdc-button/harness/mdc-button-harness.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk-experimental/testing';
+import {coerceBooleanProperty} from '@angular/cdk/coercion';
+import {ButtonHarnessFilters} from './button-harness-filters';
+
+
+/**
+ * Harness for interacting with a MDC-based mat-button in tests.
+ * @dynamic
+ */
+export class MatButtonHarness extends ComponentHarness {
+  static hostSelector = [
```

Comment:
Same as the comment above importing these

Reply:
Added a `TODO`